test.sh

```diff
@@ -1,21 +1,27 @@
 #!/usr/bin/env bash
 
-function finish {
-  echo '-----------------------test.sh------------------------------'
-  echo 'Removing test environment'
-  echo '------------------------------------------------------------'
-  docker-compose down -v
+function main() {
+#    runOSS
```

Comment:
this needs to be uncommented before merging

Reply:
💯

Reply:
👍